pyaedt/edb.py

```diff
@@ -1,20 +1,7 @@
-# -*- coding: utf-8 -*-
-"""
-This module contains all EDB functionalities in the ``Edb`` class. It inherits all objects that belong to EDB.
+"""This module contains all EDB functionalities in the ``Edb`` class. It inherits all objects that belong to EDB.
```

Comment:
I've been changing this to your recommendation, depending on the number of classes documented in the PY file:

This module contains the X class.   
This module contains these classes: X, Y, and Z.

Reply:
That works too.  User isn't going to be reading module docstrings, so it should be suited for the developer.

Reply:
```suggestion
"""This module contains the ``Edb`` class.
```